app/assets/stylesheets/_buttons.scss

```diff
@@ -1,265 +1,279 @@
-    -moz-flex-flow: $flex-dir $flex-wrap;
-    -ms-flex-flow: $flex-dir $flex-wrap;
-    flex-flow: $flex-dir $flex-wrap;
+  -webkit-flex-flow: $flex-dir $flex-wrap;
+  -moz-flex-flow: $flex-dir $flex-wrap;
+  -ms-flex-flow: $flex-dir $flex-wrap;
+  flex-flow: $flex-dir $flex-wrap;
 }
 
 @mixin justify-content($arg: flex-start) {
-    -webkit-justify-content: $arg;
-    -moz-justify-content: $arg;
-    -ms-justify-content: $arg;
-    justify-content: $arg;
+  -webkit-justify-content: $arg;
+  -moz-justify-content: $arg;
+  -ms-justify-content: $arg;
+  justify-content: $arg;
 }
 
 @mixin flex($arg: 1) {
-    -webkit-flex: $arg;
-    -moz-flex: $arg;
-    -ms-flex: $arg;
-    flex: $arg;
+  -webkit-flex: $arg;
+  -moz-flex: $arg;
+  -ms-flex: $arg;
+  flex: $arg;
 }
 
 @mixin transition-all($time) {
-    -webkit-transition: all $time ease;
-    -moz-transition: all $time ease;
-    o-transition: all $time ease;
-    transition: all $time ease;
-
+  -webkit-transition: all $time ease;
+  -moz-transition: all $time ease;
+  -o-transition: all $time ease;
+  transition: all $time ease;
 }
 
 a.button {
-    &:hover {
-       color: $medium-green;
-    }
+  &:hover {
+    color: $medium-green;
+  }
 }
 
 a.transparent-button {
-    @extend .button;
-    border-radius: $global-radius;
-    border: 1px solid $white;
-    background-color: transparent;
-    color: $white;
-    &:hover {
-        color: $white;
-        background-color: rgba(0, 0, 0, 0.4);
-    }
+  @extend .button, .large;
+  @include transition-all(100ms);
+  border-radius: $global-radius;
+  border: 2px solid $white;
+  font-weight: 500;
+  background-color: transparent;
+  color: $white;
+
+  &:hover {
+    background-color: $white;
+    color: $charcoal;
+  }
 }
 
 a.transparent-success-button {
-    @extend .button;
-    border-radius: $global-radius;
-    border: 1px solid $charcoal;
-    background-color: transparent;
-    color: $charcoal;
-    &:hover {
-        color: $medium-green;
-        border: 1px solid $medium-green;
-    }
+  @extend .button;
+  border-radius: $global-radius;
+  border: 1px solid $charcoal;
+  background-color: transparent;
+  color: $charcoal;
+
+  &:hover {
+    color: $medium-green;
+    border: 1px solid $medium-green;
+  }
 }
 
 a.success-button, input.success-button {
-    @extend .button;
-    border-radius: $global-radius;
-    border: 1px solid $white;
-    color: $white;
-    background-color: $medium-green;
-    &:hover {
-        color: $white;
-        background-color: scale-color($medium-green, $lightness: 10%);
-    }
+  @extend .button;
+  border-radius: $global-radius;
+  border: 1px solid $white;
+  color: $white;
+  background-color: $medium-green;
 
+  &:hover {
+    color: $white;
+    background-color: scale-color($medium-green, $lightness: 10%);
+  }
 }
 
 .tabs {
+  position: relative;
+  overflow: hidden;
+  margin: 0 auto;
+  width: 100%;
+  font-weight: 300;
+  font-size: 1.25em;
+
+  ul {
+    text-align: center;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered -ms-box-orient, -ms-box-pack, list-style, margin, max-width, padding, position, text-align